content/projects/gsoc/2020/projects/winsw-yaml-configs.adoc

```diff
@@ -94,8 +94,8 @@ https://github.com/winsw/winsw/releases/tag/v2.10.0[MVP has been released]. This
 * Integrate the new Windows Service Wrapper into the Jenkins core
 
 == Articles
-* link:/blog/2020/07/2020-07-08-winsw-yaml-support[GSoC 2020 - Phase 01 Updates]
-* https://www.jenkins.io/blog/2020/07/30/winsw-yaml-support-2[GSoC 2020 - Phase 02 Updates]
```

Comment:
So should this post be removed then?

Reply:
The post was removed from the repository, but nothing is removed from the actual website during deploy. This PR only makes sure the two links from [project summary](https://deploy-preview-4966--jenkins-io-site-pr.netlify.app/projects/gsoc/2020/projects/winsw-yaml-configs/) go to the posts that are present in the repo.